wazo_debug/capture.py

```diff
@@ -36,12 +37,14 @@ def _start_capture(self):
         print('Starting capture...')
         self._log_start_date()
         self._capture_logs()
-        self._capture_sip_rtp_packets()
+        self._capture_network_packets()
 
     def _stop_capture(self):
         for process in self.log_processes:
-            process.kill()
+            process.send_signal(signal.SIGINT)
             process.wait()
+            if process.returncode != 0 and process.stderr:
+                print(process.stderr.read().decode('utf8'))
```

Comment:
isn't it supposed to be `utf-8`?

Reply:
They are both equivalent. I prefer `utf-8` though, I changed it.